ksql-engine/src/test/java/io/confluent/ksql/util/QueryMetadataTest.java

```diff
@@ -131,7 +136,7 @@ public void shouldCloseKStreamsAppOnCloseThenCloseCallback() {
 
     // Then:
     final InOrder inOrder = inOrder(kafkaStreams, closeCallback);
-    inOrder.verify(kafkaStreams).close();
+    inOrder.verify(kafkaStreams).close(any(Duration.class));
```

Comment:
If the duration is to be passed in the constructor it would be nice to check the correct duration is used.

Reply:
Ack